omnigan/trainer.py

```diff
@@ -536,10 +583,16 @@ def train(self):
         for self.logger.epoch in range(
             self.logger.epoch, self.logger.epoch + self.opts.train.epochs
         ):
-            # self.infer()
             self.run_epoch()
+            self.G.eval()
+            self.D.eval()
             self.infer(verbose=1)
-            if self.logger.epoch % self.opts.train.save_n_epochs == 0:
+            self.G.train()
```

Comment:
Is it necessary to call model.train() again here if you already do it at the beginning of run_epoch() ?

Reply:
No you're right :) I think I added it inside run_epoch afterwards, by fear that if would be forgotten beforehand down the road if we re-write things. It would be a real pain in the ass to train in eval mode :p